unstructured/partition/utils/ocr_models/tesseract_ocr.py

```diff
@@ -26,21 +28,17 @@
 
 
 class OCRAgentTesseract(OCRAgent):
-    def load_agent(self):
-        pass
```

Comment:
marking since it's related to my other comment

Reply:
This `.load_agent()` implementation, even though a no-op, needed to be here because `.load_agent()` was marked as an `@abstractmethod` in the ABC. So once that was gone we no longer needed to have an implementation here.

I vaguely suspect that the original intent was to allow OCR agents to be loaded on-demand since they're a little resource-intensive. But there's no documentation to that effect or any calls to this from outside the `Paddle` implementation, so we can easily add it back in later if the design evolves that way with the addition of some other OCR platform or a change in loading strategy. Until then, this is YAGNI and dead code.